user_manual/pim/contacts.rst

```diff
@@ -272,10 +272,12 @@ Synchronizing With Android
 
 To synchronize with an Android device:
 
-1) Install CardDAV- Sync free from the Google play store by visiting `this link <https://play.google.com/store/apps/details?id=org.dmfs.carddav.sync>`_.
+1) Install CardDAV- Sync free from the Google play store by visiting `this link
+   <https://play.google.com/store/apps/details?id=org.dmfs.carddav.sync>`_.
    This app supports auto-configuration.
   
-  .. note:: Following the installing, visit carddavs://example.org/remote.php/carddav/ to auto-configure the app.
+  .. note:: Following the installing, visit carddavs://www.example.com/remote.php/dav/
```

Comment:
This could say "Following the installation..."
("installing" was already there before your formatting change, but might as well improve the grammar :)